modules/150-user-authn/crds/doc-ru-dex-authenticator.yaml

```
@@ -3,74 +3,74 @@ spec:
+                    Список адресов в формате `CIDR`, которым разрешено проходить аутентификацию. Если параметр не указан, аутентификацию разрешено проходить без ограничения по IP-адресу.
                 signOutURL:
                   description: |
-                    Передайте URL приложения, запросы с которого будут проксированы на URL выхода у `dex-authenticator`.
+                    Укажите URL приложения, с которого запросы будут перенаправляться на URL выхода в `dex-authenticator`.
                 additionalApplications:
                   description: |
-                    Список дополнительных приложений (ingress ресурсов) для которых пользовательский запрос будет перенаправлен для авторизации в Dex.
+                    Список дополнительных приложений (Ingress ресурсов), для которых пользовательский запрос будет перенаправлен для авторизации в Dex.
                   items:
                     properties:
                       domain:
-                        description: |
+                        description: |-
                           Внешний адрес вашего приложения, с которого пользовательский запрос будет перенаправлен для авторизации в Dex.
 
-                          **Не должен** содержать HTTP-схему.
+                          Адрес **не должен** содержать HTTP-схему.
                       ingressSecretName:
                         description: |
-                          Имя Secret'а с TLS-сертификатом (от домена `domain`), который используется в Ingress-объекте приложения. Secret должен обязательно находиться в том же namespace, что и DexAuthenticator.
+                          Имя Secret'а с TLS-сертификатом (от домена `domain`), который используется в Ingress-объекте приложения. Secret должен обязательно находиться в том же пространстве имен, что и DexAuthenticator.
                       ingressClassName:
                         description: |
                           Название Ingress-класса, которое будет использоваться в Ingress-объекте (должно совпадать с названием Ingress-класса для `applicationDomain` и `domain`).
                       signOutURL:
                         description: |
-                          Передайте URL приложения, запросы с которого будут проксированы на URL выхода у `dex-authenticator`.
+                          Укажите URL приложения, с которого запросы будут перенаправляться на URL выхода в `dex-authenticator`.
                       whitelistSourceRanges:
                         description: |
-                          Список адресов в формате CIDR, которым разрешено проходить аутентификацию. Если параметр не указан, аутентификацию разрешено проходить без ограничения по IP-адресу.
+                          Список адресов в формате `CIDR`, которым разрешено проходить аутентификацию. Если параметр не указан, аутентификацию разрешено проходить без ограничения по IP-адресу.
```

```suggestion
                          Список адресов в формате CIDR, которым разрешено проходить аутентификацию. Если параметр не указан, аутентификацию разрешено проходить без ограничения по IP-адресу.
```

Это распространенная аббревиатура